packages/cli/test/template.test.ts

```diff
@@ -0,0 +1,158 @@
+import { test, expect, describe, vi, afterEach } from 'vitest'
```

Comment:
These tests can be executed in the `packages/cli` directory using `pnpm test:unit` command, they use vitest to mock the commander template build command. The final test "should perform a build" will mock all points of the build, from requesting and saving a template to pushing and building with docker.